Sources/MapboxMaps/Annotations/Configuration/AnnotationOptions.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+/// :nodoc:
+public struct AnnotationSourceOptions: Equatable {
+    // TODO: Add source and cluster options
+}
+
+/// Configuration options for the AnnotationManager
+public struct AnnotationOptions: Equatable {
+    /// The identifier of the layer above the annotation layers. Default is nil, which will position the layers
+    /// above existing layers.
+    public var belowLayerId: String?
```

Comment:
why are we only account for below and not above?

Reply:
This is to align with Android - but agreed, this should probably use `LayerPosition` for both platforms.

Reply:
can we ticket that out then. I'm concerned that we only provide the one option

Reply:
I changed layer position handling in a subsequent commit. Will ticket out difference.